Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -1723,29 +1723,231 @@ public void When_non_browsable_property_differs_excluding_non_browsable_members_
             subject.Should().BeEquivalentTo(expectation, config => config.ExcludingNonBrowsableMembers());
         }
 
+        [Fact]
+        public void When_property_is_non_browsable_only_in_subject_excluding_non_browsable_members_should_make_it_succeed()
```

Comment:
🤔 Shouldn't this test fail? The `expectation` has a property that is browsable, but the equally named property on the `subject` is not-browsable and thus missing.

Reply:
No, `Exclude Non Browsable Members` doesn't mean "pretend that anything marked non-browsable doesn't exist", it means "if you see a member marked non-browsable, exclude it from the entire comparison process".

In my immediate use case, I have a legacy data type in a NuGet package that I can't update, and I have a new & better data type with mostly matching members in the codebase under test. The new & better data type includes a couple of members that the old type contains, but they aren't functional. They throw NIE and they're marked non-browsable. I can't go back in time and mark the corresponding properties in the legacy library as non-browsable.

Reply:
This is not very intuitive to me. Up to now, methods like `Including` and `Excluding` only affect the expectation, and never the subject. What do you think @jnyrup ?

Reply:
I'm pretty occupied these days, but should have time during the weekend to look more into this.

Reply:
I agree with with @dennisdoomen that only the members to select on the expectation should be affected.

Be aware that `BeEquivalentTo` is not meant to be a symmetrical assertion.

```cs
var subject = new { Important = 1, Unimportant = 2 };
var expected = new { Important = 1 };

subject.Should().BeEquivalentTo(expected);
```